<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ssmr::MainWindow</class>
 <widget class="QMainWindow" name="Ssmr::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple smart meter reader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QListWidget" name="listWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btnAddConnection">
      <property name="text">
       <string>Connection</string>
      </property>
      <property name="icon">
       <iconset resource="../ssmr.qrc">
        <normaloff>:/add.png</normaloff>:/add.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnRemoveConnection">
      <property name="text">
       <string>Connection</string>
      </property>
      <property name="icon">
       <iconset resource="../ssmr.qrc">
        <normaloff>:/delete.png</normaloff>:/delete.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <widget class="QWidget" name="pageNoConnection"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnections">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionNewConnection"/>
    <addaction name="actionRemoveConnection"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuConnections"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewConnection">
   <property name="text">
    <string>New Connection</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionRemoveConnection">
   <property name="text">
    <string>Remove Connection</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../ssmr.qrc"/>
 </resources>
 <connections/>
</ui>
